common/iputils.cc

```diff
@@ -258,6 +258,12 @@ void ComboAddress::truncate(unsigned int bits) noexcept
   *place &= (~((1<<bitsleft)-1));
 }
 
+void ComboAddress::reset()
+{
+  memset(&sin4, 0, sizeof(sin4));
```

Comment:
why not do `memset(this, 0, sizeof(*this));`

Reply:
> why not do memset(this, 0, sizeof(*this));

Because that was what was causing the compiler warning in the first place (memset of a non-trivial class). Removing the memset for sin4 should be ok I think.